source/triggers.txt

```diff
@@ -1,22 +1,105 @@
-========
-Triggers
-========
+.. _triggers:
+.. _triggers-overview:
 
-.. default-domain:: mongodb
+==============
+Atlas Triggers
+==============
 
 .. toctree::
    :titlesonly:
    :caption: Triggers
    :hidden:
 
-   About Triggers </triggers/overview>
    Database Triggers </triggers/database-triggers>
    Authentication Triggers </triggers/authentication-triggers>
    Scheduled Triggers </triggers/scheduled-triggers>
-   Usage Examples </triggers/examples>
+   Resume a Suspended Trigger </triggers/resume-a-suspended-trigger>
```

Comment:
Only DB triggers can get suspended like this, so maybe we move this procedure to the DB triggers page?